features/webxr-lighting-estimation.yml

```
@@ -0,0 +1,16 @@
+name: WebXR Lighting Estimation
+description: WebXR Lighting Estimation allows to query environmental lighting conditions within WebXR sessions.
```

I suspect something like this is going on?

```suggestion
description: The [mumble] method of the [mumble] API measures real-world environmental lighting conditions during a WebXR session.
```

---

I think it is quite a challenge to describe in a single sentence how certain APIs use different objects and methods on certain interfaces to achieve a task. Are descriptions really meant to summarize that precisely? I would need to dig into this API, but maybe a cheaper sentence like this would do, too?:

```suggestion
description: The `XRLightProbe` and `XRLightEstimate` objects contain real-world environmental lighting conditions during a WebXR session.
```

---

Yeah, I get it. I didn't understand what was meant by "allows to query" or WebXR in general, so I took a very mechanical approach. What lever do I turn to get what output?

But I think yours is very close to correct. I'd probably write it like this, to be a little pedantic.

```suggestion
description: The `XRLightProbe` and `XRLightEstimate` objects represent real-world environmental lighting conditions during a WebXR session.
```